spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js

```diff
@@ -137,12 +138,14 @@ export default class DialPad extends React.Component {
     _onGoToCall(event) {
         event.preventDefault();
 
+        analytics.log(meetingJoinEvents.DIAL_OUT);
```

Comment:
Do you think it makes sense to create actions and add the analytics part there ?